pkgs/applications/science/electronics/verilator/default.nix

```diff
@@ -12,6 +12,10 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
   buildInputs = [ perl flex_2_6_1 bison ];
 
+  postInstall = ''
+    sed -i -e '3a\#!/usr/bin/env perl' -e '1,3d' $out/bin/{verilator,verilator_coverage,verilator_profcfunc}
```

Comment:
the fixup phase handles this already, the `/usr/bin/env` usage never makes it into the final build